src/main/java/touro/simondrums/CompListenerEvents.java

```diff
@@ -0,0 +1,25 @@
+package touro.simondrums;
+
+import java.util.ArrayList;
+
+public class CompListenerEvents {
+    SimonGame game;
+    AudioPlayer audioPlayer;
+
+    public CompListenerEvents(SimonGame game, AudioPlayer audioPlayer) {
```

Comment:
prof schwimmer has mentioned in the past he doesn't like shortened words- might be better as ComputerListenerEvents